workflow-steps/install-node-modules/main.yaml

```diff
@@ -0,0 +1,19 @@
+name: Install Node Modules
+description: Installs node_modules using your configured package manager.
+
+definition:
+  using: aggregated
+  steps:
+  - name: Install Node Modules
+    script: |
+      PACKAGE_LOCK=package-lock.json
+      if test -f "$PACKAGE_LOCK"; then
```

Comment:
what's the env var `PACKAGE_LOCK`? can we not do `if test -f "package-lock.json"; then` directly?